asdf/tests/test_api.py

```diff
@@ -392,18 +392,18 @@ def test_auto_inline_masked_array(tmpdir):
         assert len(list(af.blocks.inline_blocks)) == 0
         assert len(list(af.blocks.internal_blocks)) == 2
 
-        af.write_to(outfile, auto_inline=10)
+        af.write_to(outfile, auto_inline=110)
```

Comment:
Same suggestion on computing auto_inline based on the size of the array

Reply:
I changed this to be relative to the array length.